### PR TITLE
Regenerate wrappers with Shroud-0.13.0

### PR DESCRIPTION
**Pull request by ltaylor16**
# Regenerate wrappers with Shroud-0.13.0

- This PR is a devtool update
- It does the following:
  - Part of #1258
  - `make generate`

### CODE REVIEW

File: src/axom/sidre/interface/c_fortran/wrapView.h [hunk 1]
```diff
@@ -34,164 +37,186 @@ extern "C" {
...
-                                        int NSHF_rv);
+void SIDRE_View_get_path_name_bufferify(const SIDRE_View *self,
+                                        char *SHC_rv,
+                                        int SHT_rv_len);
 
-SIDRE_Group* SIDRE_View_get_owning_group(SIDRE_View* self, SIDRE_Group* SHC_rv);
+SIDRE_Group *SIDRE_View_get_owning_group(SIDRE_View *self, SIDRE_Group *SHC_rv);
 
-bool SIDRE_View_has_buffer(const SIDRE_View* self);
+bool SIDRE_View_has_buffer(const SIDRE_View *self);
```
Comment (kennyweiss) [inline src/axom/sidre/interface/c_fortran/wrapView.h:58]: Gut check: There are lots of small formatting changes.
Did you run `make style` after `make generate`?

**Review by kennyweiss — APPROVED** [review]
Thanks @ltaylor16 !

Do we have a CI job that tests the python interface for quest? If not, did you test it manually?

File: src/axom/sidre/interface/c_fortran/wrapView.h (hunk 1 above)
Comment (ltaylor16) [reply src/axom/sidre/interface/c_fortran/wrapView.h:58]: I ran 'make style' again to check, and there were no new changes. I ran with `VERBOSE=1` to make sure that the generated files were being formatted, and they were. It's using executable at `/collab/usr/gapps/axom/devtools/toss_4_x86_64_ib/latest/llvm-10.0.0/bin/clang-format`, which seems correct.

If I edit the file to make it `Sidre_View*`, clang-format changes it to `Sidre_View *`, yet running `clang-format` on the `develop` branch preserves it as `Sidre_View*`.

Comment (kennyweiss) [reply src/axom/sidre/interface/c_fortran/wrapView.h:58]: Thanks for checking!

**Comment by ltaylor16** [comment]
I tried to compile the Python quest module and there were errors. However, they're not new errors since the develop branch is not compiling as well.  The last changes to the module were three years ago. It'll have to be fixed in another pull request.

**Comment by ltaylor16** [comment]
Can I get another approval on this? The changes are the result of `make generate` with the new Shroud.

**Review by agcapps — APPROVED** [review]

**Comment by agcapps** [comment]
Thanks, @ltaylor16 .

**Pull request merged**